docs/keycloak/manual-client-secret-setup.md

```
@@ -0,0 +1,92 @@
+3. Go to the **Clients** section from the left menu.
+4. Select the confidential client whose configuration you want to complete. ![Manage clients](./images/manage-clients.png)
+5. Go to the **Credentials** tab. ![Client credentials](./images/client-credentials.png)
+6. Generate a new credential by clicking **Regenerate secret**. ![Regenerate client secret](./images/regenerate-client-secret.png)
+7. Copy the `client_secret` value and use it in your application configuration.
+
+## 🧩 Configuring the Application to Work with Local Keycloak
+
+To enable the application to properly communicate with a locally running Keycloak instance, you need to configure the appropriate connection settings in the application.
+
+In the current project state:
+- The **`prod`** environment does not yet have the full infrastructure to operate.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)